docroot/modules/custom/foia_webform/src/FoiaSubmissionProcessingFactoryInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Drupal\foia_webform;
+
+use Drupal\node\Entity\Node;
+
+/**
+ * Provides an interface for submitting the request using the proper method.
```

Comment:
maybe instead of `using the proper method` we can say `using the component's preferred method.`